modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -66,158 +125,332 @@ The following table compares the Object Store REST API operations to Object Stor
+* <<Example: Retrieve a List of Keys in a Partition>>
+* <<Example: Retrieve the Value of a Key>>
+* <<Example: Store a Key-Value Pair>>
+* <<Example: Delete a Key and Value>>
 
-[%header,cols="10a,80a"]
-|===
-|Control Plane|Domain
-.9+|prod | object-store-us-east-2.anypoint.mulesoft.com
-| object-store-us-west-1.anypoint.mulesoft.com
-| object-store-us-west-2.anypoint.mulesoft.com
-| object-store-ap-southeast-1.anypoint.mulesoft.com
-| object-store-ap-southeast-2.anypoint.mulesoft.com
-| object-store-ap-northeast-1.anypoint.mulesoft.com
-| object-store-ca-central-1.anypoint.mulesoft.com
-| object-store-eu-west-2.eu1.anypoint.mulesoft.com
-| object-store-sa-east-1.anypoint.mulesoft.com
-.2+|prod-eu | object-store-eu-central-1.eu1.anypoint.mulesoft.com
-| object-store-eu-west-1.eu1.anypoint.mulesoft.com
-|===
+==== Example: Retrieve a List of Object Stores
 
-== View the App Client ID and Secret
+To view the list of object stores, submit a `curl` command like the following, 
+replacing BASE_URL with the regional domain, BEARER_TOKEN with your bearer token, 
+ORG_ID with your organization ID, 
+and ENV_ID with your environment ID:
 
-If you are the organization administrator for your Anypoint Platform
-account, you can view the client ID and client secret for an app from the *Object Store* menu.
+[source,console,linenums]
+----
+curl -X GET \
+-H 'authorization: bearer BEARER_TOKEN' \
+'https://BASE_URL/api/v1/organizations/ORG_ID/environments/ENV_ID/stores'
+----
 
-The client ID and secret are required to authenticate an application for use with the Object Store v2 REST API.
+This command returns output like the following:
 
-. Log into *Anypoint Platform > Runtime Manager*.
-. Click an application that has Object Store access and click *Manage Application*. The app can be running or undeployed.
-. If the app is not yet associated with Object Store v2, click the checkbox and apply changes.
-. Click the *Object Store* menu in the left navigation area, then click *Show Client Credentials*.
-. Click the *Copy to Clipboard* icon, and paste the client ID or secret value into your curl application that you use to authenticate the REST API.
+[source,json,linenums]
+----
+  {"values":[{"storeId":"APP_os-simple-flow__defaultPersistentObjectStore","encrypted":true,"permanentOsFlag":false,
+    "persistent":true,"defaultTtlSeconds":2592000,"defaultConfirmationTtlSeconds":600,"isFixedTtl":null},
+  {"storeId":"APP_scheduler-tests__defaultPersistentObjectStore","encrypted":true,"permanentOsFlag":false,
+    "persistent":true,"defaultTtlSeconds":2592000,"defaultConfirmationTtlSeconds":600,"isFixedTtl":null}
+  {"storeId":"APP_sending-a-csv-file-thru-email-using-smtp__defaultPersistentObjectStore","encrypted":true,"permanentOsFlag":false,
+    "persistent":true,"defaultTtlSeconds":2592000,"defaultConfirmationTtlSeconds":600,"isFixedTtl":null}],"nextPageToken":null}
+----
+
+Use the `storeID` in place of STORE_ID in the examples that follow. 
```

Comment:
This requires the audience to have looked at the first example and not skip down to one more to their purposes, so it risks not being seen....
```suggestion
Use `storeID` in place of STORE_ID in the examples that follow. 
```